src/app/utils/createElement.ts

```diff
@@ -0,0 +1,32 @@
+import { elementConfig } from '../interfaces/elementConfig'
+
+export function createElement({
+  type,
+  content,
+  id,
+  classes,
+  handleEvent,
+  properties,
+  innerHTML,
+  styles,
+}: elementConfig) {
+  const element = document.createElement(type)
+  if (content) {
+    element.textContent = content
+  }
+  if (id) {
+    element.id = id
+  }
+  if (classes) {
+    classes.forEach((className) => {
+      element.className += `${className} `
+    })
+  }
+  if (handleEvent?.type) {
+    element.addEventListener(handleEvent.type, () => handleEvent.callback())
+  }
```

Comment:
What if I would like to add more event listeners? Maybe you can use here an array?